### PR TITLE
Updated pdfjs-dist

### PR DESCRIPTION
**Pull request by spencerAJ**
Latest versions of pdfjs-dist caused worker loader errors either importing the package or when trying to set the workerSrc. Used a diff on [this](https://github.com/Submitty/pdf-annotate.js) fork to come up with these fixes which puts pdfjs-dist at v2.4.456. 
Pdfjs does have prerelease versions which include [this](https://github.com/mozilla/pdf.js/commit/1ff578cbd417cf3ff4979255bb04b4f28dbdd322) fix which we think may resolve the issue in the future.

### CODE REVIEW

File: docs/shared/pdf_viewer.css [hunk 1]
```diff
@@ -24,23 +24,18 @@
   line-height: 1.0;
 }
 
-.textLayer > div {
+.textLayer > span {
   color: transparent;
   position: absolute;
   white-space: pre;
   cursor: text;
-  -webkit-transform-origin: 0% 0%;
```
Comment (srphillips) [inline docs/shared/pdf_viewer.css:32]: I know the regular one is still in place, but the removal of these `transform-origin` styles, especially on the div itself may be what's causing the misalignment we keep seeing.

File: package.json [hunk 2]
```diff
@@ -23,38 +20,41 @@
   "author": "Matt Zabriskie",
```
Comment (srphillips) [inline package.json]: We need to add us as an author and make reference to the original author and fork in the docs/Readme.

File: shared/pdf_viewer.css [hunk 3]
```diff
@@ -24,23 +24,18 @@
   line-height: 1.0;
 }
 
-.textLayer > div {
+.textLayer > span {
```
Comment (srphillips) [inline shared/pdf_viewer.css:27]: Again, adjustment that would remove the `transform-origin` from the `.textLayer` div I think.

**Review by srphillips — CHANGES_REQUESTED** [review]
Make sure we note if we pulled anything directly from another repository.  So if you directly implemented a change from another repo, note where the changes came from with a link in a comment block in the file.

**Review by srphillips — APPROVED** [review]

**Comment by srphillips** [comment]
I've approved this, but I still think there's some CSS trickery here that might be causing our layer mis-alignment.  Let's check with @ThinkBrandonThink about if this what's causing the issue.

**Comment by spencerAJ** [comment]
> I've approved this, but I still think there's some CSS trickery here that might be causing our layer mis-alignment. Let's check with @ThinkBrandonThink about if this what's causing the issue.

I looked into this issue with Brandon and found that the what was causing the mis-alignment was due to the style sheet not being properly loaded into the iframe. The easiest solution was to copy the style sheet into the project in adhesion and make minor adjustments there.

File: docs/shared/pdf_viewer.css (hunk 1 above)
Comment (spencerAJ) [reply docs/shared/pdf_viewer.css:32]: Added in [this](https://github.com/atomicjolt/pdf-annotate.js/pull/1/commits/5d803a82c4e603886538b1b7858e31659e754ab1) commit

**Comment by rogercorrea** [comment]
Do you have a code with implementation using the updated to PdfJs 2.4 version?
I have fork of 2019 but I tried update my code using your PdfAnnotate version but I'm with erros.

Example:
My version use:
`annotationLayer = PDFJSAnnotate.UI.preparePage();`

Your version doesn't work because:
caught TypeError: Cannot read properties of undefined (reading 'UI')

**Pull request merged**